common-module/src/main/java/domain/model/Tag.java

```diff
@@ -0,0 +1,38 @@
+package domain.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import org.hibernate.annotations.CreationTimestamp;
+import domain.status.UserTag;
+
+@Entity
+public class Tag {
+
+  @Id
+  @GeneratedValue
+  private long id;
+
+  @OneToOne(fetch = FetchType.LAZY)
+  private User user;
+
+  @Enumerated(value = EnumType.STRING)
+  private UserTag name;
+
+  @Column(nullable = false, updatable = false)
+  @CreationTimestamp
+  private LocalDateTime createdAt;
+  private LocalDateTime updatedAt;
+
+  @OneToMany(fetch = FetchType.LAZY)
+  private List<Board> members = new ArrayList<>();
```

Comment:
Board 엔티티에 있는 ```tag``` 를 매핑해줘야 하므로 ```mappedBy``` 추가해주시고,
변수 네이밍도 ```members``` 가 아니라 ```boards``` 가 괜찮을 거 같네요.
```suggestion
  @OneToMany(fetch = FetchType.LAZY, mappedBy = "tag")
  private List<Board> boards = new ArrayList<>();
```